x-pack/plugins/cases/public/common/navigation/paths.test.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  getCreateCasePath,
+  getSubCaseViewPath,
+  getCaseViewPath,
+  getCasesConfigurePath,
+  getCaseViewWithCommentPath,
+  getSubCaseViewWithCommentPath,
+  generateCaseViewPath,
+} from './paths';
+
+describe('Paths', () => {
+  describe('getCreateCasePath', () => {
+    it('returns the correct path', () => {
+      expect(getCreateCasePath('test')).toBe('test/create');
+    });
+  });
+
+  describe('getCasesConfigurePath', () => {
+    it('returns the correct path', () => {
+      expect(getCasesConfigurePath('test')).toBe('test/configure');
+    });
+  });
+
+  describe('getCaseViewPath', () => {
+    it('returns the correct path', () => {
+      expect(getCaseViewPath('test')).toBe('test/:detailName');
+    });
+  });
+
+  describe('getSubCaseViewPath', () => {
+    it('returns the correct path', () => {
+      expect(getSubCaseViewPath('test')).toBe('test/:detailName/sub-cases/:subCaseId');
+    });
+  });
+
+  describe('getCaseViewWithCommentPath', () => {
+    it('returns the correct path', () => {
+      expect(getCaseViewWithCommentPath('test')).toBe('test/:detailName/:commentId');
+    });
+  });
+
+  describe('getSubCaseViewWithCommentPath', () => {
+    it('returns the correct path', () => {
+      expect(getSubCaseViewWithCommentPath('test')).toBe(
+        'test/:detailName/sub-cases/:subCaseId/:commentId'
+      );
+    });
+  });
+
+  describe('generateCaseViewPath', () => {
+    it('returns the correct path', () => {
+      expect(generateCaseViewPath({ detailName: 'my-case' })).toBe('/my-case');
+    });
+
+    it('returns the correct path when subCaseId is provided', () => {
+      expect(generateCaseViewPath({ detailName: 'my-case', subCaseId: 'my-sub-case' })).toBe(
+        '/my-case/sub-cases/my-sub-case'
```

Comment:
@semd Interesting. I didn't know we have the concept of subcases. How can I view subcases in the UI? And btw shouldn't the link be `'/cases/my-case/sub-cases/my-sub-case'`. I thought so, since the cases details page is under `/cases/my-case` path

Reply:
Yes, there is no way to create sub-cases any longer, but we kept the implementation by now, we will probably clean everything at once eventually.
But the routes are still supported:
https://github.com/elastic/kibana/pull/117582/files#diff-70de957354013a49ef572e305d933e88236d2b258df2413b5ef83ad30a60b42bR81-R84